releasenotes/releasenotes.rst

```diff
@@ -35,12 +56,15 @@ Database
 .. attention::
    For SQL Server we changed the datatype of the primary keys. The related upgradescript (`data/20210519072216_ChangePrimaryKeyTypeFromIntToBigint.sql`) can take a lot of time if you have many resources loaded in your database. Therefore some guidelines:
 
-   * We tested it on a database with about 15k Patient records, and 14 mln resources in total. Migrating that took about 45 minutes on a fairly powerful laptop.
+   * We tested it on a database with about 15k Patient records, and 14 mln resources in total. Migrating that took about 50 minutes on a fairly powerful laptop.
    * Absolutely make sure you create a backup of your database first.
    * If you haven't done so already, first upgrade to version 4.1.x.
    * If you already expect the migration might time out, you can run it manually upfront. Shut down Firely Server, so no other users are using the database, and then run the script from SQL Server Management Studio (or a similar tool).
    * Running the second script (`20210520102224_ChangePrimaryKeyTypeFromIntToBigintBDE.sql`) is optional - that should also succeed when applied by the automigration.
 
+.. note::
+   As reported by a client, for this version of Firely Server, some foreign keys are untrusted in the SQL Server database. This has an impact on query performance and referential integrity. If you find yourself in the same situation, please use the script provided in version 4.2.1 to fix the issue.
+
```

Comment:
I have moved this section all the way to the top of 4.2.0 release notes.

Reply:
Thanks a lot Lennart, the wordings are much better. I've remove this "note" section.